Patches.py

```diff
@@ -2290,7 +2290,35 @@ def update_scrub_text(message: bytearray, text_replacement: list[str], default_p
     if world.settings.correct_chest_appearances == 'both':
         symbol = rom.sym('CHEST_SIZE_TEXTURE')
         rom.write_int32(symbol, 0x00000001)
-        # Move Ganon's Castle's Zelda's Lullaby Chest back so is reachable if large
+
+    # Specify which textures we want
+    symbol = rom.sym('CHEST_GILDED_TEXTURE')
+    rom.write_byte(symbol, 0x01)
+    if 'major' not in world.settings.chest_textures_specific:
+        rom.write_byte(symbol, 0x00)
```

Comment:
This can be simplified to:

```suggestion
    rom.write_byte(rom.sym('CHEST_GILDED_TEXTURE'), 'major' in world.settings.chest_textures_specific)
```

Same for the others.